packages/vite-plugin-checker/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import chalk from 'chalk'
 import { spawn } from 'child_process'
-import pick from 'lodash.pick'
+import pick from 'lodash/pick'
```

Comment:
I don't know why, but this helps to run the tests. 

This code is valid anyway.

```suggestion
import pick from 'lodash/pick.js'
```

Reply:
it's seems working with this

Reply:
Thanks for investigating! I've committed the change. Not sure anyone will merge this pull request, though :shrug:

Reply:
it's because of typescript configuration about import of default exported module